### ISSUE TITLE
Is the input to the model that is point cloud data supported?

### ISSUE THREAD

**Issue by auniquesun** [zhijian-liu/torchprofile#25]
@zhijian-liu As the title shows, I want to count the #params of a point cloud model (e.g., [PointBERT](https://github.com/Julie-tang00/Point-BERT)) instead of an image model, the example input of point cloud data is of shape `(batch_size, 1024, 3)`. It means each point cloud has 1024 points with 3 coordinates, can I achieve this goal by calling the API in you library?

Thanks.

**Comment by zhijian-liu** [comment]
Yes — TorchProfile works with any model that `torch.jit.trace` supports, regardless of input modality. Point cloud inputs work the same way:

```python
import torch
from torchprofile import profile_macs

model = YourPointCloudModel().eval()
inputs = torch.randn(1, 1024, 3)  # (batch, num_points, coords)
macs = profile_macs(model, inputs)
```

Note that `profile_macs` counts MACs (multiply-accumulate operations), not parameters. For parameter count, use `sum(p.numel() for p in model.parameters())`.
